examples/restify/client.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const api = require('@opentelemetry/api');
+const tracer = require('./tracer')('example-http-client');
+// eslint-disable-next-line import/order
+const http = require('http');
+
+/** A function which makes requests and handles response. */
+function makeRequest(path) {
+  // span corresponds to outgoing requests. Here, we have manually created
+  // the span, which is created to track work that happens outside of the
+  // request lifecycle entirely.
+  const span = tracer.startSpan(`makeRequest to ${path}`);
+  api.context.with(api.setSpan(api.context.active(), span), () => {
```

Comment:
Do we need to include manual span creation in the example? I think we should keep the client as simple as possible for an example focusing on server/restify instrumentation. HTTP instrumentation should generate a client span anyway.

Reply:
True. It would make sense if the HTTP instrumentation was not there. I'll remove that.